app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  def create
+    @talk = Talk.find(params[:talk_id])
+    @comment = Comment.new(talk_id: @talk.id, content: params[:comment][:content], user_id: 1)
+    if @comment.save
+      flash[:notice] = 'Thanks for commenting'
+    else
+    @comment.errors.full_messages.each do |error|
+      flash[:alert] = "Comment can't be blank"
+    end
+    end
+    redirect_to talk_path(@talk)
+  end
+
+  def edit
+    @comment = Comment.find(params[:id])
+    @talk = @comment.talk
+  end
+
+  def update
+    @comment = Comment.find(params[:id])
+    @comment.content = params[:comment][:content]
+    if @comment.save
+      flash[:notice] = 'Thanks for commenting'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.